server.js

```diff
@@ -1,17 +1,43 @@
+require('newrelic');
 const express = require('express');
 const morgan = require('morgan');
 const path = require('path');
 const cors = require('cors');
 
+const React = require('react')
+
+const ReactDOMServer = require ('react-dom/server');
+
+const Page  = require('./htmlTemplates/page.js')
+const Body =  require('./htmlTemplates/body.js')
+const Scripts =  require('./htmlTemplates/scripts.js')
+const rawComponents = require('./bundles.js')
```

Comment:
I found rawComponents to be a confusing name for what these actually are (the component classes).